minard/views.py

```diff
@@ -740,18 +740,20 @@ def calibdq():
 @app.route('/calibdq_tellie')
 def calibdq_tellie():
     run_dict = {}
-    run_numbers = HLDQTools.import_TELLIE_runnumbers()
+    limit = request.args.get("limit", 10, type=int)
+    offset = request.args.get("offset", 0, type=int)
+    run_numbers = HLDQTools.import_TELLIE_runnumbers(limit=10,offset=offset)
```

Comment:
Should pass limit here if you want to be able to change the number of runs displayed.